src/tests/unit/tests/common/components/theme.test.tsx

```diff
@@ -43,39 +45,29 @@ describe('ThemeInner', () => {
     test.each(testStub)('is high contrast mode enabled: %s', (enableHighContrast: boolean) => {
         props.storeState.userConfigurationStoreData.enableHighContrast = enableHighContrast;
         const renderResult = render(<ThemeInner {...props} />);
-
-        loadThemeMock.mockReset(); // omits irrelevant mock-call records from snapshot
         expect(renderResult.asFragment()).toMatchSnapshot();
         expectMockedComponentPropsToMatchSnapshots([BodyClassModifier]);
     });
 
     test.each(testStub)(
         'componentDidMount: is high contrast mode enabled: %s',
         (enableHighContrast: boolean) => {
-            const theme = enableHighContrast ? HighContrastTheme : DefaultTheme;
             const userConfigurationStoreData = { enableHighContrast } as UserConfigurationStoreData;
             render(<ThemeInner {...props} storeState={{ userConfigurationStoreData }} />);
-
-            expect(loadThemeMock).toHaveBeenCalledWith(theme);
+            const hasThemeProp = getMockComponentClassPropsForCall(ThemeProvider).theme;
```

Comment:
should we be checking for both the v8 theme and the v9 theme here?

Reply:
Hi @madalynrose agree, added check for v9 as well now, please re-review. Thanks